.circleci/config.yml

```diff
@@ -14,23 +14,23 @@ commands:
             find . -type f -name 'yarn.lock' -not -path "*node_modules*" -exec cat
             {} + >> ~/.tmp/checksumfiles/yarn.lock
           name: Create cache checksum file
-      - restore_cache:
-          keys:
-            - >-
-              yarn-cache-{{ arch }}-{{ checksum "~/.tmp/checksumfiles/package.json"
-              }}-{{ checksum "~/.tmp/checksumfiles/yarn.lock" }}-{{
-              .Environment.CACHE_VERSION }}
+      # - restore_cache:
+      #     keys:
+      #       - >-
+      #         yarn-cache-{{ arch }}-{{ checksum "~/.tmp/checksumfiles/package.json"
+      #         }}-{{ checksum "~/.tmp/checksumfiles/yarn.lock" }}-{{
+      #         .Environment.CACHE_VERSION }}
```

Comment:
This step is failing, still don't know why. I have tried by using `/tmp/yarn` and `./tmp/yarn` with the same results. 

The orb has an open issue about it, so I will monitor and see if they fix it https://github.com/react-native-community/react-native-circleci-orb/issues/66, until the, I think we can live without the caches

Reply:
let's add a comment and a link to https://github.com/react-native-community/react-native-circleci-orb/issues/66 to the code here, so that when we look back at it we understand _why_ it's commented and we check if it has been fixed